index.d.ts

```diff
@@ -0,0 +1,112 @@
+/** Declaration file generated by dts-gen */
+
+export = dog_statsy;
+
+declare class dog_statsy {
+    constructor(...args: any[]);
```

Comment:
The constructor options should be typed: https://github.com/segmentio/dog-statsy#clientopts